common/url.go

```diff
@@ -53,16 +53,19 @@ const (
 )
 
 var (
+	// DubboNodes ...
 	DubboNodes = [...]string{"consumers", "configurators", "routers", "providers"}
 	DubboRole  = [...]string{"consumer", "", "", "provider"}
```

Comment:
comment for DubboRole

Reply:
not hinted by golint. guess a bug of golint. add comment in next commit.